src/theme/components/input.ts

```diff
@@ -0,0 +1,37 @@
+const Input = {
+  Input: {
+    sizes: {
+      md: {
+        field: {
+          fontSize: '17px',
+          px: '12px',
+          h: '56px',
+          lineHeight: '1.2941',
+          borderRadius: '2lg',
```

Comment:
Show the border radius here be `12px`? 💭 

Current style | The design 
--- | ---
![Screen Shot 2565-05-30 at 17 34 58](https://user-images.githubusercontent.com/1772999/170974726-0e74a7ec-d94c-49be-84f5-f7b3b343480d.png) | ![Screen Shot 2565-05-30 at 17 33 29](https://user-images.githubusercontent.com/1772999/170974573-0c3c3bc2-b4d8-4926-a9ba-433ec94f5789.png)

Reply:
I fixed it by using `xl` because the default theme has that one.
https://github.com/Lahphim/react-surveyor-challenge/pull/53/commits/a503ca3881e4081dc33140806bf6800d495d1fcc

![Default_Theme_-_Chakra_UI](https://user-images.githubusercontent.com/749672/170984143-a412f0bc-fa5d-45e6-bb3d-432d0eb6c936.png)